### ISSUE TITLE
Model in training mode during demo.py

### ISSUE THREAD

**Issue by braydenrudisill** [qinzheng93/GeoTransformer#116]
Model.eval() needs to be added in the demo.py for the model be in evaluation mode. Being stuck in training mode enables this line of code which uses the ground truth transformation. 

```python
# 7 Random select ground truth node correspondences during training
if self.training:
    ref_node_corr_indices, src_node_corr_indices, node_corr_scores = self.coarse_target(
        gt_node_corr_indices, gt_node_corr_overlaps
    )
```

**Comment by poppo0214** [comment]
Hi, I am facing the same problem.
How can I add "Model.eval()" in the demo.py??

**Comment by cduch9** [comment]
Thanks a lot! It would have saved me a lot of time if I had seen this post earlier.
